pom.xml

```diff
@@ -93,6 +93,7 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <external.build.directory>${project.basedir}/target</external.build.directory>
+    <external.createDependencyReducedPom>true</external.createDependencyReducedPom>
```

Comment:
Please add some commentary here. It should indicate (a) why this property exists and (b) that any build requiring poms must not alter the value as the installed pom will be incorrect.